_packaging/gaphor.spec

```diff
@@ -87,14 +83,14 @@ a = Analysis(  # type: ignore
     hooksconfig={
         "gi": {
             "module-versions": {
-                "Gtk": "3.0" if os.getenv("GAPHOR_PKG_GTK") == "3" else "4.0",
-                "GtkSource": "4" if os.getenv("GAPHOR_PKG_GTK") == "3" else "5",
+                "Gtk": "4.0",
+                "GtkSource": "5",
             },
         },
     },
     hookspath=["."],
     runtime_hooks=[
-        f"use_gtk_{os.getenv('GAPHOR_PKG_GTK', '4')}.py",
+        f"use_gtk_4.py",
```

Comment:
I think we can remove this whole runtime hook since `GAPHOR_USE_GTK=4` is the default.